llvm/lib/Target/AMDGPU/AMDGPU.h

```diff
@@ -39,6 +39,8 @@ FunctionPass *createSIFoldOperandsLegacyPass();
 FunctionPass *createSIPeepholeSDWALegacyPass();
 FunctionPass *createSILowerI1CopiesLegacyPass();
 FunctionPass *createAMDGPUGlobalISelDivergenceLoweringPass();
+FunctionPass *createAMDGPURBSelectPass();
```

Comment:
Please spell out RegBank wherever possible, it makes it easier to grep for things in the codebase.

Reply:
Sure, will wait for more comments then bulk update everything

Reply:
Do you have any suggestions since AMDGPURegBankSelect is taken? 
AMDGPURBSelect -> ... 
AMDGPURBLegalize -> AMDGPURegBankLegalize is fine

Reply:
Idk, I was hoping we could get rid of the other one :) Maybe we could call this one AMDGPUStandaloneRegBankSelect, since it's not inheriting from RegBankSelect (at least if that's the intention for the long term).

Reply:
The AMDGPURegBankSelect is functional except for some functions with divergent control flow. Deleting it now breaks almost all globalisel regression tests.
When new regbankselect path gets functional we can make it a default path and decide what to do with AMDGPURegBankSelect, maybe rename it to legacy or just delete it.

New pass names as suggested:
 AMDGPUStandaloneRegBankSelect
 AMDGPURegBankLegalize

Reply:
You can delete the old one and replace it with the generic regbankselect pass. It's not doing anything in the pass other than running the analysis

Reply:
Reverting default reg-bank-select to RegBankSelect, new pass will take
AMDGPURegBankSelect